[2013-01-28 21:06:55] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 21:06:55] cinch.INFO: < 302 [] []
[2013-01-28 21:06:55] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:06:56] cinch.CRITICAL: Twig_Error_Loader: Template "login.html" is not defined (). (uncaught exception) at /var/www/cinch/cinch/vendor/twig/twig/lib/Twig/Loader/Chain.php line 113 [] []
[2013-01-28 21:06:56] cinch.INFO: exception 'Twig_Error_Loader' with message 'Template "login.html" is not defined ().' in /var/www/cinch/cinch/vendor/twig/twig/lib/Twig/Loader/Chain.php:113
Stack trace:
#0 /var/www/cinch/cinch/vendor/twig/twig/lib/Twig/Environment.php(266): Twig_Loader_Chain->getCacheKey('login.html')
#1 /var/www/cinch/cinch/vendor/twig/twig/lib/Twig/Environment.php(313): Twig_Environment->getTemplateClass('login.html', NULL)
#2 /var/www/cinch/cinch/vendor/twig/twig/lib/Twig/Environment.php(289): Twig_Environment->loadTemplate('login.html')
#3 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application/TwigTrait.php(64): Twig_Environment->render('login.html', Array)
#4 /var/www/cinch/cinch/app/Cinch/Controller/AdminController.php(50): Cinch\Application->renderView('login.html', Array)
#5 [internal function]: Cinch\Controller\AdminController->login(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Array, Array)
#7 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#8 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#9 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#10 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#11 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#12 {main} [] []
[2013-01-28 21:06:56] cinch.INFO: < 500 [] []
[2013-01-28 21:18:40] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:18:40] cinch.INFO: < 200 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /css/audioplayerv1.min.css" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /css/style.css" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /css/video-js.min.css" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/video.min.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/modernizr.custom.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /css/fancybox.min.css" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/jquery.easing-1.3.min.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/jquery.fancybox.pack.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/respond.min.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/jquery.smartStartSlider.min.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/jquery.jcarousel.min.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/jquery.isotope.min.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/jquery.cycle.all.min.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/jquery.touchSwipe.min.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/audioplayerv1.min.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:40] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/jquery.gmap.min.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:41] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /img/logo.png" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:18:41] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /js/custom.js" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 21:22:13] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:22:13] cinch.INFO: < 200 [] []
[2013-01-28 21:22:59] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:22:59] cinch.INFO: < 200 [] []
[2013-01-28 21:24:19] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:24:19] cinch.INFO: < 200 [] []
[2013-01-28 21:24:42] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:24:42] cinch.INFO: < 200 [] []
[2013-01-28 21:26:31] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:26:31] cinch.INFO: < 200 [] []
[2013-01-28 21:27:03] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:27:03] cinch.INFO: < 200 [] []
[2013-01-28 21:27:20] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:27:20] cinch.INFO: < 200 [] []
[2013-01-28 21:28:20] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:28:20] cinch.INFO: < 200 [] []
[2013-01-28 21:30:11] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:30:11] cinch.INFO: < 200 [] []
[2013-01-28 21:31:33] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:31:33] cinch.INFO: < 200 [] []
[2013-01-28 21:36:23] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:36:23] cinch.INFO: < 200 [] []
[2013-01-28 21:37:34] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:37:34] cinch.INFO: < 200 [] []
[2013-01-28 21:37:56] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:37:56] cinch.INFO: < 200 [] []
[2013-01-28 21:38:29] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:38:29] cinch.INFO: < 200 [] []
[2013-01-28 21:40:13] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:40:13] cinch.INFO: < 200 [] []
[2013-01-28 21:40:32] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:40:32] cinch.INFO: < 200 [] []
[2013-01-28 21:41:04] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:41:04] cinch.INFO: < 200 [] []
[2013-01-28 21:41:37] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:41:37] cinch.INFO: < 200 [] []
[2013-01-28 21:42:00] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:42:00] cinch.INFO: < 200 [] []
[2013-01-28 21:42:57] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:42:57] cinch.INFO: < 200 [] []
[2013-01-28 21:54:35] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 21:54:35] cinch.INFO: < 200 [] []
[2013-01-28 22:00:26] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 22:00:27] cinch.INFO: < 200 [] []
[2013-01-28 22:01:18] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /img/loader.gif" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 22:01:18] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "POST /php/contact-send.php" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 22:05:05] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 22:05:05] cinch.INFO: < 200 [] []
[2013-01-28 22:05:25] cinch.INFO: < 302 [] []
[2013-01-28 22:05:25] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 22:05:25] cinch.INFO: < 200 [] []
[2013-01-28 22:07:13] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 22:07:13] cinch.INFO: < 200 [] []
[2013-01-28 22:07:38] cinch.INFO: > GET /cinch/public/index_dev.php/about [] []
[2013-01-28 22:07:38] cinch.INFO: < 200 [] []
[2013-01-28 22:07:48] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 22:07:48] cinch.INFO: < 302 [] []
[2013-01-28 22:07:48] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 22:07:48] cinch.INFO: < 200 [] []
[2013-01-28 22:09:14] cinch.INFO: < 302 [] []
[2013-01-28 22:09:14] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 22:09:14] cinch.INFO: < 200 [] []
[2013-01-28 22:10:02] cinch.INFO: < 302 [] []
[2013-01-28 22:10:02] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 22:10:02] cinch.INFO: < 200 [] []
[2013-01-28 22:12:07] cinch.INFO: > GET /cinch/public/index_dev.php/login [] []
[2013-01-28 22:12:07] cinch.INFO: < 200 [] []
[2013-01-28 22:12:11] cinch.INFO: < 302 [] []
[2013-01-28 22:12:11] cinch.INFO: > GET /cinch/public/index_dev.php/admin [] []
[2013-01-28 22:12:11] cinch.INFO: < 301 [] []
[2013-01-28 22:12:11] cinch.INFO: > GET /cinch/public/index_dev.php/admin/ [] []
[2013-01-28 22:12:11] cinch.INFO: < 301 [] []
[2013-01-28 22:12:11] cinch.INFO: > GET /cinch/public/index_dev.php/admin/dashboard [] []
[2013-01-28 22:12:11] cinch.INFO: < 200 [] []
[2013-01-28 22:12:20] cinch.INFO: > GET /cinch/public/index_dev.php/logout [] []
[2013-01-28 22:12:20] cinch.CRITICAL: LogicException: The controller must be a callable (Array(0 => Object(Cinch\Controller\AdminController), 1 => process_logout) given). (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/Event/FilterControllerEvent.php line 68 [] []
[2013-01-28 22:12:20] cinch.INFO: exception 'LogicException' with message 'The controller must be a callable (Array(0 => Object(Cinch\Controller\AdminController), 1 => process_logout) given).' in /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/Event/FilterControllerEvent.php:68
Stack trace:
#0 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/Event/FilterControllerEvent.php(42): Symfony\Component\HttpKernel\Event\FilterControllerEvent->setController(Array)
#1 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(121): Symfony\Component\HttpKernel\Event\FilterControllerEvent->__construct(Object(Symfony\Component\HttpKernel\HttpKernel), Array, Object(Symfony\Component\HttpFoundation\Request), 1)
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#3 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#5 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#7 {main} [] []
[2013-01-28 22:12:20] cinch.INFO: < 500 [] []
[2013-01-28 22:21:46] cinch.INFO: < 302 [] []
[2013-01-28 22:21:46] cinch.INFO: > GET /cinch/public/index_dev.php/ [] []
[2013-01-28 22:21:46] cinch.INFO: < 200 [] []
[2013-01-28 22:21:57] cinch.INFO: > GET /cinch/public/index_dev.php/logout [] []
[2013-01-28 22:21:57] cinch.CRITICAL: LogicException: The controller must be a callable (Array(0 => Object(Cinch\Controller\AdminController), 1 => process_logout) given). (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/Event/FilterControllerEvent.php line 68 [] []
[2013-01-28 22:21:57] cinch.INFO: exception 'LogicException' with message 'The controller must be a callable (Array(0 => Object(Cinch\Controller\AdminController), 1 => process_logout) given).' in /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/Event/FilterControllerEvent.php:68
Stack trace:
#0 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/Event/FilterControllerEvent.php(42): Symfony\Component\HttpKernel\Event\FilterControllerEvent->setController(Array)
#1 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(121): Symfony\Component\HttpKernel\Event\FilterControllerEvent->__construct(Object(Symfony\Component\HttpKernel\HttpKernel), Array, Object(Symfony\Component\HttpFoundation\Request), 1)
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#3 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#5 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#7 {main} [] []
[2013-01-28 22:21:57] cinch.INFO: < 500 [] []
[2013-01-28 22:22:00] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /logoutd" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 22:22:34] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /logout" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 22:24:21] cinch.INFO: > GET /cinch/public/index_dev.php/ [] []
[2013-01-28 22:24:21] cinch.INFO: < 200 [] []
[2013-01-28 22:33:39] cinch.INFO: > GET /cinch/public/ [] []
[2013-01-28 22:33:39] cinch.INFO: < 200 [] []
[2013-01-28 22:35:01] cinch.INFO: > GET /cinch/public/ [] []
[2013-01-28 22:35:01] cinch.INFO: < 200 [] []
[2013-01-28 22:35:09] cinch.INFO: > GET /cinch/public/ [] []
[2013-01-28 22:35:09] cinch.INFO: < 200 [] []
[2013-01-28 22:35:19] cinch.INFO: > GET /cinch/public/ [] []
[2013-01-28 22:35:19] cinch.INFO: < 200 [] []
[2013-01-28 22:53:10] cinch.INFO: > GET /cinch/public/index.php/about [] []
[2013-01-28 22:53:10] cinch.INFO: < 200 [] []
[2013-01-28 22:55:06] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 22:55:06] cinch.INFO: < 200 [] []
[2013-01-28 22:55:44] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 22:55:44] cinch.INFO: < 200 [] []
[2013-01-28 22:55:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 22:55:52] cinch.INFO: < 302 [] []
[2013-01-28 22:55:52] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 22:55:52] cinch.INFO: < 200 [] []
[2013-01-28 22:56:01] cinch.INFO: < 302 [] []
[2013-01-28 22:56:01] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 22:56:01] cinch.INFO: < 200 [] []
[2013-01-28 22:56:05] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 22:56:05] cinch.INFO: < 200 [] []
[2013-01-28 22:56:10] cinch.INFO: < 302 [] []
[2013-01-28 22:56:10] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 22:56:10] cinch.INFO: < 200 [] []
[2013-01-28 22:56:15] cinch.INFO: < 302 [] []
[2013-01-28 22:56:15] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 22:56:15] cinch.INFO: < 200 [] []
[2013-01-28 22:56:19] cinch.INFO: < 302 [] []
[2013-01-28 22:56:19] cinch.INFO: > GET /cinch/public/admin [] []
[2013-01-28 22:56:19] cinch.INFO: < 301 [] []
[2013-01-28 22:56:19] cinch.INFO: > GET /cinch/public/admin/ [] []
[2013-01-28 22:56:19] cinch.INFO: < 301 [] []
[2013-01-28 22:56:19] cinch.INFO: > GET /cinch/public/admin/dashboard [] []
[2013-01-28 22:56:19] cinch.INFO: < 200 [] []
[2013-01-28 22:56:25] cinch.INFO: < 302 [] []
[2013-01-28 22:56:25] cinch.INFO: > GET /cinch/public/ [] []
[2013-01-28 22:56:25] cinch.INFO: < 200 [] []
[2013-01-28 22:57:01] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 22:57:01] cinch.INFO: < 200 [] []
[2013-01-28 22:57:03] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 22:57:03] cinch.INFO: < 200 [] []
[2013-01-28 22:57:08] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 22:57:08] cinch.INFO: < 200 [] []
[2013-01-28 22:58:26] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 22:58:26] cinch.INFO: < 200 [] []
[2013-01-28 23:01:30] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 23:01:30] cinch.INFO: < 200 [] []
[2013-01-28 23:01:36] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:01:36] cinch.INFO: < 302 [] []
[2013-01-28 23:01:36] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 23:01:36] cinch.INFO: < 200 [] []
[2013-01-28 23:02:06] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:02:07] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:02:41] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /index_dev.phpabout" (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 92 [] []
[2013-01-28 23:02:44] cinch.INFO: > GET /cinch/public/index_dev.php/about [] []
[2013-01-28 23:02:44] cinch.CRITICAL: ErrorException: Notice: Undefined property: Cinch\Provider\SiteControllerProvider::$app in /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 52 (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 52 [] []
[2013-01-28 23:02:44] cinch.INFO: exception 'ErrorException' with message 'Notice: Undefined property: Cinch\Provider\SiteControllerProvider::$app in /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 52' in /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php:52
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\HttpKernel\Debug\ErrorHandler->handle(8, 'Undefined prope...', '/var/www/cinch/...', 52, Array)
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:02:44] cinch.INFO: < 500 [] []
[2013-01-28 23:03:19] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:03:19] cinch.CRITICAL: ErrorException: Notice: Undefined property: Cinch\Provider\SiteControllerProvider::$app in /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 52 (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 52 [] []
[2013-01-28 23:03:19] cinch.INFO: exception 'ErrorException' with message 'Notice: Undefined property: Cinch\Provider\SiteControllerProvider::$app in /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 52' in /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php:52
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\HttpKernel\Debug\ErrorHandler->handle(8, 'Undefined prope...', '/var/www/cinch/...', 52, Array)
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:03:19] cinch.INFO: < 500 [] []
[2013-01-28 23:03:40] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:03:41] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:03:41] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:03:41] cinch.INFO: < 500 [] []
[2013-01-28 23:09:20] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:09:20] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:09:20] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:09:20] cinch.INFO: < 500 [] []
[2013-01-28 23:09:21] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:09:21] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:09:21] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:09:21] cinch.INFO: < 500 [] []
[2013-01-28 23:09:22] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:09:22] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:09:22] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:09:22] cinch.INFO: < 500 [] []
[2013-01-28 23:09:23] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:09:23] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:09:23] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:09:23] cinch.INFO: < 500 [] []
[2013-01-28 23:09:41] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:09:41] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:09:41] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:09:41] cinch.INFO: < 500 [] []
[2013-01-28 23:12:35] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:12:35] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:12:35] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:12:35] cinch.INFO: < 500 [] []
[2013-01-28 23:13:11] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:13:11] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:13:11] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:13:11] cinch.INFO: < 500 [] []
[2013-01-28 23:13:14] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:13:14] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:13:14] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:13:14] cinch.INFO: < 500 [] []
[2013-01-28 23:13:15] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:13:15] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:13:15] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:13:15] cinch.INFO: < 500 [] []
[2013-01-28 23:13:15] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:13:15] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:13:15] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:13:15] cinch.INFO: < 500 [] []
[2013-01-28 23:13:15] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:13:15] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:13:15] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:13:15] cinch.INFO: < 500 [] []
[2013-01-28 23:13:15] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:13:15] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:13:15] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:13:15] cinch.INFO: < 500 [] []
[2013-01-28 23:13:16] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:13:16] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:13:16] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:13:16] cinch.INFO: < 500 [] []
[2013-01-28 23:13:16] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:13:16] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:13:16] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(52): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:13:16] cinch.INFO: < 500 [] []
[2013-01-28 23:21:27] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:21:27] cinch.INFO: < 302 [] []
[2013-01-28 23:21:27] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 23:21:27] cinch.INFO: < 200 [] []
[2013-01-28 23:21:32] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:21:32] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AccessDeniedException: Access Denied (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 53 [] []
[2013-01-28 23:21:32] cinch.INFO: < 302 [] []
[2013-01-28 23:21:32] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 23:21:32] cinch.INFO: < 200 [] []
[2013-01-28 23:22:38] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 23:22:38] cinch.INFO: < 200 [] []
[2013-01-28 23:22:41] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:22:41] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 54 [] []
[2013-01-28 23:22:50] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 23:22:50] cinch.INFO: < 200 [] []
[2013-01-28 23:22:53] cinch.INFO: < 302 [] []
[2013-01-28 23:22:53] cinch.INFO: > GET /cinch/public/admin [] []
[2013-01-28 23:22:53] cinch.INFO: < 301 [] []
[2013-01-28 23:22:53] cinch.INFO: > GET /cinch/public/admin/ [] []
[2013-01-28 23:22:53] cinch.INFO: < 301 [] []
[2013-01-28 23:22:53] cinch.INFO: > GET /cinch/public/admin/dashboard [] []
[2013-01-28 23:22:53] cinch.INFO: < 200 [] []
[2013-01-28 23:22:57] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:22:57] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 54 [] []
[2013-01-28 23:22:59] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:22:59] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 54 [] []
[2013-01-28 23:23:00] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:23:00] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 54 [] []
[2013-01-28 23:23:00] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:23:00] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 54 [] []
[2013-01-28 23:23:01] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:23:01] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 54 [] []
[2013-01-28 23:23:01] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:23:01] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 54 [] []
[2013-01-28 23:23:01] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:23:01] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 54 [] []
[2013-01-28 23:23:01] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:23:01] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 54 [] []
[2013-01-28 23:23:02] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:23:02] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 54 [] []
[2013-01-28 23:23:39] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:23:39] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 54 [] []
[2013-01-28 23:23:44] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:23:44] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 54 [] []
[2013-01-28 23:24:15] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:24:24] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:33:51] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:51] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:52] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:52] cinch.INFO: < 302 [] []
[2013-01-28 23:33:53] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:53] cinch.INFO: < 302 [] []
[2013-01-28 23:33:53] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:53] cinch.INFO: < 302 [] []
[2013-01-28 23:33:53] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:53] cinch.INFO: < 302 [] []
[2013-01-28 23:33:53] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:53] cinch.INFO: < 302 [] []
[2013-01-28 23:33:53] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: A Token was not found in the SecurityContext. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Http/Firewall/AccessListener.php line 53 [] []
[2013-01-28 23:33:53] cinch.INFO: < 302 [] []
[2013-01-28 23:34:56] cinch.INFO: > GET /cinch/public/login [] []
[2013-01-28 23:34:56] cinch.INFO: < 200 [] []
[2013-01-28 23:40:30] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:40:30] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:40:30] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(51): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:40:30] cinch.INFO: < 500 [] []
[2013-01-28 23:40:36] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:40:36] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:40:36] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(51): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:40:36] cinch.INFO: < 500 [] []
[2013-01-28 23:41:00] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:41:00] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:41:00] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(51): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:41:00] cinch.INFO: < 500 [] []
[2013-01-28 23:41:07] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:41:07] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:41:07] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(51): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:41:07] cinch.INFO: < 500 [] []
[2013-01-28 23:41:09] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:41:09] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:41:09] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(51): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:41:09] cinch.INFO: < 500 [] []
[2013-01-28 23:41:09] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:41:09] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:41:09] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(51): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:41:09] cinch.INFO: < 500 [] []
[2013-01-28 23:41:09] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:41:09] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:41:09] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(51): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:41:09] cinch.INFO: < 500 [] []
[2013-01-28 23:41:10] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:41:10] cinch.CRITICAL: Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException: The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL. (uncaught exception) at /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php line 61 [] []
[2013-01-28 23:41:10] cinch.INFO: exception 'Symfony\Component\Security\Core\Exception\AuthenticationCredentialsNotFoundException' with message 'The security context contains no authentication token. One possible reason may be that there is no firewall configured for this URL.' in /var/www/cinch/cinch/vendor/symfony/security/Symfony/Component/Security/Core/SecurityContext.php:61
Stack trace:
#0 /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php(51): Symfony\Component\Security\Core\SecurityContext->isGranted('ROLE_ADMIN')
#1 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#2 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#3 /var/www/cinch/cinch/vendor/symfony/http-kernel/Symfony/Component/HttpKernel/HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#4 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#5 /var/www/cinch/cinch/vendor/silex/silex/src/Silex/Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#6 /var/www/cinch/cinch/app/bootstrap.php(135): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#7 /var/www/cinch/public/index_dev.php(39): require_once('/var/www/cinch/...')
#8 {main} [] []
[2013-01-28 23:41:10] cinch.INFO: < 500 [] []
[2013-01-28 23:41:23] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:41:23] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 53 [] []
[2013-01-28 23:58:18] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:58:18] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 52 [] []
[2013-01-28 23:58:52] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:58:52] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 52 [] []
[2013-01-28 23:58:54] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:58:54] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 52 [] []
[2013-01-28 23:58:55] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:58:55] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 52 [] []
[2013-01-28 23:58:55] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:58:55] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 52 [] []
[2013-01-28 23:58:55] cinch.INFO: > GET /cinch/public/about [] []
[2013-01-28 23:58:55] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at /var/www/cinch/cinch/app/Cinch/Provider/SiteControllerProvider.php line 52 [] []
[2013-02-06 01:13:42] cinch.INFO: > GET /cinch/public/about [] []
[2013-02-06 01:13:42] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at C:\xampp\htdocs\cinch\cinch\app\Cinch\Provider\SiteControllerProvider.php line 52 [] []
[2013-02-06 01:14:12] cinch.INFO: > GET /cinch/public/about [] []
[2013-02-06 01:14:12] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at C:\xampp\htdocs\cinch\cinch\app\Cinch\Provider\SiteControllerProvider.php line 52 [] []
[2013-02-06 01:14:46] cinch.INFO: > GET /cinch/public/about [] []
[2013-02-06 01:14:46] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at C:\xampp\htdocs\cinch\cinch\app\Cinch\Provider\SiteControllerProvider.php line 52 [] []
[2013-02-06 01:14:46] cinch.INFO: < 404 [] []
[2013-02-06 01:15:00] cinch.INFO: > GET /cinch/public/about [] []
[2013-02-06 01:15:00] cinch.CRITICAL: Twig_Error_Loader: Template "@Cinch/templates/column.html" is not defined () in "/about-us.html" at line 114. (uncaught exception) at C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Loader\Chain.php line 113 [] []
[2013-02-06 01:15:00] cinch.INFO: exception 'Twig_Error_Loader' with message 'Template "@Cinch/templates/column.html" is not defined () in "/about-us.html" at line 114.' in C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Loader\Chain.php:113
Stack trace:
#0 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(266): Twig_Loader_Chain->getCacheKey('@Cinch/template...')
#1 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(313): Twig_Environment->getTemplateClass('@Cinch/template...', NULL)
#2 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(289): Twig_Environment->loadTemplate('@Cinch/template...')
#3 C:\xampp\htdocs\cinch\cinch\app\Cinch\Block.php(68): Twig_Environment->render('@Cinch/template...', Array)
#4 C:\xampp\htdocs\cinch\cinch\packages\Cinch\Blocks\Column.php(27): Cinch\Block->render('@Cinch/template...')
#5 C:\xampp\htdocs\cinch\cinch\app\Cinch\Region.php(49): Cinch\Blocks\Column->display()
#6 C:\xampp\htdocs\cinch\cinch\app\Cinch\TwigExtension.php(85): Cinch\Region->display()
#7 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(321) : eval()'d code(166): Cinch\TwigExtension->regionFunction(Object(Twig_Environment), 'main')
#8 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Template.php(265): __TwigTemplate_3caf47857318dfc8f76152510a798d15->doDisplay(Array, Array)
#9 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Template.php(239): Twig_Template->displayWithErrorHandling(Array, Array)
#10 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Template.php(250): Twig_Template->display(Array)
#11 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(289): Twig_Template->render(Array)
#12 C:\xampp\htdocs\cinch\cinch\vendor\silex\silex\src\Silex\Application\TwigTrait.php(64): Twig_Environment->render('/about-us.html', Array)
#13 C:\xampp\htdocs\cinch\cinch\app\Cinch\Provider\SiteControllerProvider.php(55): Cinch\Application->renderView('/about-us.html', Array)
#14 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#15 C:\xampp\htdocs\cinch\cinch\vendor\symfony\http-kernel\Symfony\Component\HttpKernel\HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#16 C:\xampp\htdocs\cinch\cinch\vendor\symfony\http-kernel\Symfony\Component\HttpKernel\HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#17 C:\xampp\htdocs\cinch\cinch\vendor\silex\silex\src\Silex\Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#18 C:\xampp\htdocs\cinch\cinch\vendor\silex\silex\src\Silex\Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#19 C:\xampp\htdocs\cinch\cinch\app\bootstrap.php(118): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#20 C:\xampp\htdocs\cinch\public\index_dev.php(39): require_once('C:\xampp\htdocs...')
#21 {main} [] []
[2013-02-06 01:15:00] cinch.INFO: < 500 [] []
[2013-02-06 01:19:21] cinch.INFO: > GET /cinch/public/about [] []
[2013-02-06 01:19:21] cinch.CRITICAL: Twig_Error_Loader: Template "@Cinch/templates/column.html" is not defined () in "/about-us.html" at line 114. (uncaught exception) at C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Loader\Chain.php line 113 [] []
[2013-02-06 01:19:21] cinch.INFO: exception 'Twig_Error_Loader' with message 'Template "@Cinch/templates/column.html" is not defined () in "/about-us.html" at line 114.' in C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Loader\Chain.php:113
Stack trace:
#0 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(266): Twig_Loader_Chain->getCacheKey('@Cinch/template...')
#1 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(313): Twig_Environment->getTemplateClass('@Cinch/template...', NULL)
#2 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(289): Twig_Environment->loadTemplate('@Cinch/template...')
#3 C:\xampp\htdocs\cinch\cinch\app\Cinch\Block.php(68): Twig_Environment->render('@Cinch/template...', Array)
#4 C:\xampp\htdocs\cinch\cinch\packages\Cinch\Blocks\Column.php(27): Cinch\Block->render('@Cinch/template...')
#5 C:\xampp\htdocs\cinch\cinch\app\Cinch\Region.php(49): Cinch\Blocks\Column->display()
#6 C:\xampp\htdocs\cinch\cinch\app\Cinch\TwigExtension.php(85): Cinch\Region->display()
#7 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(321) : eval()'d code(166): Cinch\TwigExtension->regionFunction(Object(Twig_Environment), 'main')
#8 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Template.php(265): __TwigTemplate_3caf47857318dfc8f76152510a798d15->doDisplay(Array, Array)
#9 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Template.php(239): Twig_Template->displayWithErrorHandling(Array, Array)
#10 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Template.php(250): Twig_Template->display(Array)
#11 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(289): Twig_Template->render(Array)
#12 C:\xampp\htdocs\cinch\cinch\vendor\silex\silex\src\Silex\Application\TwigTrait.php(64): Twig_Environment->render('/about-us.html', Array)
#13 C:\xampp\htdocs\cinch\cinch\app\Cinch\Provider\SiteControllerProvider.php(55): Cinch\Application->renderView('/about-us.html', Array)
#14 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#15 C:\xampp\htdocs\cinch\cinch\vendor\symfony\http-kernel\Symfony\Component\HttpKernel\HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#16 C:\xampp\htdocs\cinch\cinch\vendor\symfony\http-kernel\Symfony\Component\HttpKernel\HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#17 C:\xampp\htdocs\cinch\cinch\vendor\silex\silex\src\Silex\Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#18 C:\xampp\htdocs\cinch\cinch\vendor\silex\silex\src\Silex\Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#19 C:\xampp\htdocs\cinch\cinch\app\bootstrap.php(118): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#20 C:\xampp\htdocs\cinch\public\index_dev.php(39): require_once('C:\xampp\htdocs...')
#21 {main} [] []
[2013-02-06 01:19:21] cinch.INFO: < 500 [] []
[2013-02-06 01:20:10] cinch.INFO: > GET /cinch/public/about [] []
[2013-02-06 01:20:10] cinch.INFO: < 200 [] []
[2013-02-06 01:21:24] cinch.INFO: > GET /cinch/public/about [] []
[2013-02-06 01:21:24] cinch.INFO: < 200 [] []
[2013-02-06 01:21:34] cinch.INFO: > GET /cinch/public/ [] []
[2013-02-06 01:21:34] cinch.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException:  (uncaught exception) at C:\xampp\htdocs\cinch\cinch\app\Cinch\Provider\SiteControllerProvider.php line 52 [] []
[2013-02-06 01:21:34] cinch.INFO: < 404 [] []
[2013-02-06 01:22:00] cinch.INFO: > GET /cinch/public/about [] []
[2013-02-06 01:22:00] cinch.CRITICAL: Twig_Error_Loader: Template "@cinch/templates/column.html" is not defined (). (uncaught exception) at C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Loader\Chain.php line 113 [] []
[2013-02-06 01:22:00] cinch.INFO: exception 'Twig_Error_Loader' with message 'Template "@cinch/templates/column.html" is not defined ().' in C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Loader\Chain.php:113
Stack trace:
#0 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(266): Twig_Loader_Chain->getCacheKey('@cinch/template...')
#1 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(313): Twig_Environment->getTemplateClass('@cinch/template...', NULL)
#2 C:\xampp\htdocs\cinch\cinch\vendor\twig\twig\lib\Twig\Environment.php(289): Twig_Environment->loadTemplate('@cinch/template...')
#3 C:\xampp\htdocs\cinch\cinch\vendor\silex\silex\src\Silex\Application\TwigTrait.php(64): Twig_Environment->render('@cinch/template...', Array)
#4 C:\xampp\htdocs\cinch\cinch\app\Cinch\Provider\SiteControllerProvider.php(55): Cinch\Application->renderView('@cinch/template...', Array)
#5 [internal function]: Cinch\Provider\SiteControllerProvider->Cinch\Provider\{closure}()
#6 C:\xampp\htdocs\cinch\cinch\vendor\symfony\http-kernel\Symfony\Component\HttpKernel\HttpKernel.php(129): call_user_func_array(Object(Closure), Array)
#7 C:\xampp\htdocs\cinch\cinch\vendor\symfony\http-kernel\Symfony\Component\HttpKernel\HttpKernel.php(73): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#8 C:\xampp\htdocs\cinch\cinch\vendor\silex\silex\src\Silex\Application.php(479): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#9 C:\xampp\htdocs\cinch\cinch\vendor\silex\silex\src\Silex\Application.php(456): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request))
#10 C:\xampp\htdocs\cinch\cinch\app\bootstrap.php(118): Silex\Application->run(Object(Symfony\Component\HttpFoundation\Request))
#11 C:\xampp\htdocs\cinch\public\index_dev.php(39): require_once('C:\xampp\htdocs...')
#12 {main} [] []
[2013-02-06 01:22:00] cinch.INFO: < 500 [] []
[2013-02-06 01:22:29] cinch.INFO: > GET /cinch/public/about [] []
[2013-02-06 01:22:30] cinch.INFO: < 200 [] []
[2013-02-06 01:22:33] cinch.INFO: > GET /cinch/public/ [] []
[2013-02-06 01:22:33] cinch.INFO: < 200 [] []
